blank-canvas-blocks/sass/blocks/_navigation.scss

```diff
@@ -54,7 +55,7 @@
 	}
 }
 
-@include media(mobile-only) {
+@media (max-width: #{ ($break-mobile) }) {
```

Comment:
Should this be a mixin like on line 48? If Gutenberg doesn't have it, maybe we want to make one using this variable?

Reply:
Gutenberg doesn't have a mixin for this. IMO the important thing it to be using the same breakpoints. Happy to add a mixin if you see value in it...

Reply:
I'm just thinking in case we need to use it multiple times, just for consistency and to type a bit less. So far we use it just once, but that may change

Reply:
Yea, I'm there too.  My 'druthers would be to keep the mixins we already have and adjusting them to leverage Gutenberg's values (though renaming them to less silly names sounds nice while we have the chance).

Reply:
I don't see the value in keeping our own mixing when we can use Gutenberg's ones. If you look at the diff for this change, I'm not convinced that the mixin adds much, but if we do add one we should only do it for the cases that Gutenberg doesn't cover.

Reply:
I didn't like the idea of importing external dependencies anywhere we might want to use a Media Query and instead keep those external dependencies in a single place (_breakpoints.scss). Gutenberg provides the values that BCB leverages in the media-query mixin.

I too wouldn't want to provide/leverage any breakpoints that weren't derived from Gutenberg's breakpoint values.

Really though it's only one location, isn't likely to be many more and don't feel very strongly about it; just a 'druthers.